LiveChat/components/Chat.js

```diff
@@ -134,20 +40,25 @@ export default class Chat extends React.Component {
     if (this.props.isChatOn) {
       return (
         <View
-          animation="lightSpeedIn"
+          animation="fadeInUp"
           style={styles.container}
           ref={(ref) => { this.chat = ref; }}
         >
           <NavigationBar chatTitle={this.props.chatTitle} closeChat={this.closeChat} />
-          <Text style={styles.status}>
-            { this.state.onlineStatus ? this.props.greeting : this.props.noAgents }
-          </Text>
+          { this.props.connectionState !== 'connected' && <Text style={styles.connectionStatus}>Reconnecting...</Text> }
```

Comment:
I would suggest extracting this expression to a variable at the beginning of a render method:
```js
const isReconnecting = this.props.connectionState !== 'connected'
```

And then here just use it:
```suggestion
          {isReconnecting && <Text style={styles.connectionStatus}>Reconnecting...</Text>}
```